portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,93 @@
+let answer = 0;
+
+function loadInTruth() {
+
+    let firstTruth = Math.floor(Math.random() * truths.length);
+    let secondTruth = Math.floor(Math.random() * truths.length);
+    while (firstTruth == secondTruth && truths.length != 1) {
+        secondTruth = Math.floor(Math.random() * truths.length);
+    }
+
+    let firstPlace = Math.floor(Math.random() * 3);
+    let secondPlace = Math.floor(Math.random() * 3);
+    while (firstPlace == secondPlace) {
+        secondPlace = Math.floor(Math.random() * 3);
+    }
+
+    let truthLocations = new Array(2);
+    truthLocations[0] = firstPlace;
+    truthLocations[1] = secondPlace;
```

Comment:
Since this isn't used until you return it, just declare it at the end of the method.

Even better...

"return new Array(firstPlace, secondPlace)"

Reply:
Changed return to end and to be more intuitive